src/frontend/js/components/LanguageSelector/index.tsx

```diff
@@ -62,9 +63,7 @@ const LanguageSelector: FC<LanguageSelectorProps> = ({ currentLanguage, language
       </label>
       <button {...getToggleButtonProps()} className="selector__button">
         {selectedItem?.name || intl.formatMessage(messages.languages)}
-        <svg role="img" className="selector__button__icon" aria-hidden="true">
-          <use xlinkHref="#icon-chevron-down" />
-        </svg>
+        <Icon name="icon-chevron-down" className="selector__button__icon" />
```

Comment:
This icon has now the `.icon` className applied but it introduces a visual regression.

<img width="81" alt="image" src="https://user-images.githubusercontent.com/9265241/155690251-04318268-deff-4d71-b130-3986b5ac91c3.png">

Reply:
Oh, I was definitely not awake yesterday when open this PR then... There were other few spots where the "icon" class wasn't used and I didn't use the new component there. Felt weird to have the choice to "disable" internal behavior via a new prop so I figured not using the component in those cases was better. Will update